packages/react-widgets/src/widgets/GeocoderWidget.js

```diff
@@ -53,12 +53,17 @@ const SearchIcon = (args) => (
  * @param  {object} props
  * @param  {Object} [props.className] - Material-UI withStyle class for styling
  * @param  {Function} [props.onError] - Function to handle error messages from the widget.
+ * @param  {Object=} [props.credentials] - Credentials to use, otherwise widgets checks redus state.
+ * @param  {string=} props.method - method, either `sql` or `lds`, see geocodeStreetPoint
+ * @param  {Boolean=} [props.zoomControl] - Optional, default true. Whether control should zoom map on result.
```

Comment:
When I read `zoomControl` I though that it would add a zoom control with `+` and `-` 😆  Maybe `zoomToResult: boolean` (like the fn) would be clearer?

Reply:
Aggree